Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/signature/core/DscSignatureValidatorTest.kt

```diff
@@ -0,0 +1,24 @@
+package de.rki.coronawarnapp.covidcertificate.signature.core
```

Comment:
We should also test this against our minAPI. Encryption stuff often has API related issues.